### PR TITLE
Akka Streams

### PR DESCRIPTION
**Pull request by ignasi35**
References #59

I am not very inspired. So this is a collage from contents in https://doc.akka.io/docs/akka/current/stream/stream-introduction.html

### CODE REVIEW

File: docs-source/docs/modules/concepts/pages/akka-stream.adoc [hunk 1]
```diff
@@ -6,4 +6,16 @@
 include::partial$include.adoc[]
 
 
-ifdef::todo[TODO: this page is WIP]
+{akka}/stream/stream-introduction.html[Akka Streams] provides an intuitive and safe way to formulate stream processing setups such that we can then execute them efficiently and with bounded resource usage —- no more OutOfMemoryErrors.
+
+The Akka Streams API is completely decoupled from the xref:reactive-strams.adoc[] interfaces. While Akka Streams focus on the formulation of transformations on data streams the scope of Reactive Streams is to define a common mechanism of how to move data across an asynchronous boundary without losses, buffering or resource exhaustion.
```
Comment (patriknw) [inline docs-source/docs/modules/concepts/pages/akka-stream.adoc:11]: reactive-strams renamed in other PR

File: docs-source/docs/modules/concepts/pages/akka-stream.adoc [hunk 2]
```diff
@@ -6,4 +6,16 @@
 include::partial$include.adoc[]
 
 
-ifdef::todo[TODO: this page is WIP]
+{akka}/stream/stream-introduction.html[Akka Streams] provides an intuitive and safe way to formulate stream processing setups such that we can then execute them efficiently and with bounded resource usage —- no more OutOfMemoryErrors.
+
+The Akka Streams API is completely decoupled from the xref:reactive-strams.adoc[] interfaces. While Akka Streams focus on the formulation of transformations on data streams the scope of Reactive Streams is to define a common mechanism of how to move data across an asynchronous boundary without losses, buffering or resource exhaustion. 
+
+The relationship between these two is that the Akka Streams API is geared towards end-users while the Akka Streams implementation uses the Reactive Streams interfaces internally to pass data between the different operators.
```
Comment (patriknw) [inline docs-source/docs/modules/concepts/pages/akka-stream.adoc:13]: maybe highlight here also that Akka Streams implements Reactive Streams and is fully compatible with other implementations

File: docs-source/docs/modules/concepts/pages/akka-stream.adoc [hunk 3]
```diff
@@ -6,4 +6,16 @@
...
+
+To learn more, have a look at the {akka}/stream/stream-introduction.html[Akka Streams documentation] dedicated setions: 
+
+* Read the {akka}/stream/stream-quickstart.html[Quick Start Guide] to get a feel for how streams look like and what they can do.
+* Read the {akka}/general/stream/stream-design.html[Design Principles behind Akka Streams] for a top down learning approach.
+* Read the {akka}/stream/stream-cookbook.html[Streams Cookbook] for a hands-on approach.
```
Comment (patriknw) [inline docs-source/docs/modules/concepts/pages/akka-stream.adoc:19]: not sure the cookbook is a good starting point for learning more

File: docs-source/docs/modules/concepts/pages/akka-stream.adoc [hunk 4]
```diff
@@ -6,4 +6,16 @@
...
+* Have a look at the {akka}/stream/operators/index.html[operator index] for a complete overview of the built-in processing operators.
+* Check the {akka}/stream/stream-flows-and-basics.html[reference guide] for an in-depth documentationinto specific topics.
```
Comment (patriknw) [inline docs-source/docs/modules/concepts/pages/akka-stream.adoc:21]: typo documentationinto

File: docs-source/docs/modules/concepts/pages/akka-stream.adoc (hunk 1 above)
Comment (ignasi35) [reply docs-source/docs/modules/concepts/pages/akka-stream.adoc:11]: Good catch (I trusted the autocomplete when adding this link)

File: docs-source/docs/modules/concepts/pages/akka-stream.adoc (hunk 3 above)
Comment (ignasi35) [reply docs-source/docs/modules/concepts/pages/akka-stream.adoc:19]: I'll remove it. I copy/pasted the list items from the ref docs.

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**